packaging/linux/Arch/PKGBUILD

```diff
@@ -69,43 +114,74 @@ prepare() {
 build() {
     export BRANCH="@GITHUB_BRANCH@"
     export BUILD_VERSION="@BUILD_VERSION@"
-    export COMMIT="@GITHUB_COMMIT@"
+    export COMMIT="${_commit}"
 
     export CC="gcc-${_gcc_version}"
     export CXX="g++-${_gcc_version}"
 
     export CFLAGS="${CFLAGS/-Werror=format-security/}"
     export CXXFLAGS="${CXXFLAGS/-Werror=format-security/}"
 
-    cmake \
-        -S "$pkgname" \
-        -B build \
-        -Wno-dev \
-        -D BUILD_DOCS=OFF \
-        -D BUILD_WERROR=ON \
-        -D CMAKE_INSTALL_PREFIX=/usr \
-        -D SUNSHINE_EXECUTABLE_PATH=/usr/bin/sunshine \
-        -D SUNSHINE_ASSETS_DIR="share/sunshine" \
-        -D SUNSHINE_PUBLISHER_NAME='LizardByte' \
-        -D SUNSHINE_PUBLISHER_WEBSITE='https://app.lizardbyte.dev' \
-        -D SUNSHINE_PUBLISHER_ISSUE_URL='https://app.lizardbyte.dev/support'
+    export CUDA_PATH=/opt/cuda
+    export NVCC_CCBIN="/usr/bin/g++-$_cuda_gcc_version"
+
+    export MAKEFLAGS="-j$(nproc)"
```

Comment:
Should we rely on the user's `MAKEFLAGS` env variable instead of overwriting it?

According to https://wiki.archlinux.org/title/Makepkg#Parallel_compilation make uses this automatically. Am I missing something?

Reply:
I've modified it so that it only sets it when not present